code/modules/vehicles/motorbike.dm

```diff
@@ -195,8 +197,7 @@
 /obj/vehicle/ridden/motorbike/projectile_hit(obj/projectile/P)
 	if(!buckled_mobs)
 		return ..()
-	var/mob/buckled_mob = pick(buckled_mobs)
-	return buckled_mob.projectile_hit(P)
+	return FALSE
```

Comment:
```suggestion
	return src == proj.original_target
```
How about that? Just a suggestion though

Reply:
I like it

Reply:
Set whoever is buckled as the src.